mobile/src/components/common/content/content.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import { useWindowDimensions } from 'react-native';
+import RenderHtml from 'react-native-render-html';
+
+import { sanitizeHTML } from '~/helpers/helpers';
+
+import { defaultTagsStyles, styles } from './styles';
+
+type Props = {
+  html: string;
+  contentWidth?: number;
+};
+
+const Content: FC<Props> = ({ html, contentWidth }) => {
+  const { width } = useWindowDimensions();
```

Comment:
Remove useWindowDimensions from this component and also import it from '~/hooks/hooks'